blog/views.py

```diff
@@ -0,0 +1,57 @@
+from django.http import HttpResponse
+from django.shortcuts import render
+from django.utils import timezone
+
+from blog.models import Reminder
+from blog.forms import ReminderForm
+
+
+def index(request):
+    return render(request, "blog/index.html", {})
+
+
+def add(request):
+    r = Reminder()
```

Comment:
Don't give variable names like this. Try giving a meaningful name. You can use reminder as variable in this scenario.